src/appleseed.cli/CMakeLists.txt

```diff
@@ -59,11 +59,11 @@ add_executable (appleseed.cli
     ${appleseed.cli_sources}
 )
 
-if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+if (UNIX AND NOT WIN32 AND NOT APPLE)
```

Comment:
François, with all due respect, but did you read the commit log? :-)  I've explicitly explained why `if (UNIX)` alone it not enough.